Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+# Dataset Addition Procedures
+
+First of all, thank you for sharing your data with CONP and the scientific community!
+
+All datasets to upload to the CONP portal must include the following files in the root directory:
+
+- a required `README.md` file. The content of this file will be displayed in the portal page describing your dataset.
+- a required `DATS.json` file as described in [the main documentation page](https://github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). We provide a [DATS GUI editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. Note: the content of this file will also be displayed on the portal page describing your dataset.
+- optionally, a study/institutional logo to be displayed with your dataset on the portal.
+
+It is possible to upload a dataset using one of the following options:
+
+- [I. Upload using Zenodo](#zenodo)
+- [II. Upload using OSF](#osf)
+- [III. Upload on the CONP FTP server](#conp-server)
+- [IV. Upload using DataLad](#datalad)
+
+If you need help at any stage, please [open an issue in
+the CONP-PCNO/conp-dataset repository](https://github.com/CONP-PCNO/conp-dataset/issues/new/choose) and we will do our best to help you.
+
+## <a name="zenodo"></a> I. Upload using Zenodo
+
+1. Prepare your dataset for upload. **Please note the following requirements:**
+
+	- All datasets must include a `README.md` file in the root directory. That file will be displayed on the CONP portal page that will describe the dataset.
+	- All datasets must include a `DATS.json` file as described in [the main documentation page](https://	github.com/CONP-PCNO/conp-documentation/blob/master/CONP_DATS_fields.md). The content of this file will 	also be displayed on the CONP portal page that will describe the dataset. We provide a [DATS GUI 	editor]() ***(#TODO: ADD LINK TO EDITOR)*** for easy creation of this file. 
```

Comment:
Same comment applies here, also what's up with the spacing between "will" and "also"?

Reply:
thanks! I will fix the spacing!